src/http/request.ts

```diff
@@ -147,6 +148,30 @@ export class Request extends ServerRequest {
     return this.headers.get(input);
   }
 
+  /**
+   * Gets all header params
+   *
+   * @return Key value pairs for the header name and it's value
```

Comment:
```suggestion
   * @returns Key value pairs for the header name and it's value
```

Reply:
seems @returns isn't something the docs support, but @return is: https://doc.deno.land/https/deno.land/x/sockets@v0.5.0/mod.ts

Reply:
wow that's garbage.... tsdoc states `@returns` -_-. alright then